build/StablePackagesInfo.props

```diff
@@ -0,0 +1,14 @@
+<Project>
+    <ItemGroup>
+        <StableProject Include="Microsoft.ML"/>
+        <StableProject Include="Microsoft.ML.CpuMath"/>
+        <StableProject Include="Microsoft.ML.FastTree"/>
+        <StableProject Include="Microsoft.ML.ImageAnalytics"/>
+        <StableProject Include="Microsoft.ML.LightGbm"/>
+        <StableProject Include="Microsoft.ML.Mkl.Components"/>
+        <StableProject Include="Microsoft.ML.Mkl.Redist"/>
+    </ItemGroup>
+    <PropertyGroup>
+        <StableProjects>@(StableProject, ';')</StableProjects>
```

Comment:
Ha ha, that's one way I guess!

Reply:
Does this actually work? Properties in MSBuild are evaluated before Items. Which means when the `StableProjects` property gets defined, `@(StableProject)` will be empty.

---
In reply to: [265677312](https://github.com/dotnet/machinelearning/pull/2951#discussion_r265677312) [](ancestors = 265677312)

Reply:
To fix this, you can just define

```xml
<PropertyGroup>
    <StableProjects>
        Microsoft.ML;
        Microsoft.ML.CpuMath;
        Microsoft.ML.FastTree;
    ....
    </StableProjects>
</PropertyGroup>
```

---
In reply to: [265710272](https://github.com/dotnet/machinelearning/pull/2951#discussion_r265710272) [](ancestors = 265710272,265677312)